Examples/ObjectiveC/InsetMapExample.m

```diff
@@ -0,0 +1,71 @@
+#import "InsetMapExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleInsetMap = @"InsetMapExample";
+
+@interface InsetMapExample () <MGLMapViewDelegate>
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) MGLMapView *miniMapView;
+@end
+
+@implementation InsetMapExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    // Set the map's center coordinate
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(59.31, 18.06)
+                zoomLevel:9
+                 animated:NO];
+    
+    self.miniMapView = [[MGLMapView alloc] initWithFrame:CGRectZero];
+    [self.miniMapView setAllowsScrolling:NO];
+    [self.miniMapView setAllowsTilting:NO];
+    [self.miniMapView setAllowsZooming:NO];
+    [self.miniMapView setAllowsRotating:NO];
+
+    [self.miniMapView.logoView setHidden:YES];
+    self.miniMapView.attributionButton.tintColor = UIColor.clearColor;
+     UIColor *black = [UIColor blackColor];
+    [self.miniMapView.layer setBorderColor: black.CGColor];
+    [self.miniMapView.layer setBorderWidth:1];
+    
+    [self.miniMapView setCenterCoordinate:self.mapView.centerCoordinate zoomLevel:self.mapView.zoomLevel animated:NO];
+    [self.miniMapView setTranslatesAutoresizingMaskIntoConstraints:NO];
+
+    // Set the delegate property of our map view to self after instantiating it
+    self.mapView.delegate = self;
+
+    [self.view addSubview:self.mapView];
+    [self.view addSubview:_miniMapView];
```

Comment:
Please use the `self.property` syntax instead of `_property`. This is more consistent with the rest of our examples. If you're curious as to what the difference is, here is the [short explanation](https://teamtreehouse.com/community/objc-underscore-self-notation-difference#:~:text=When%20you%20have%20an%20object,calling%20the%20instance%20variable%20directly.) and here's the [long explanation](https://useyourloaf.com/blog/understanding-your-objective-c-self/).